tests/settings/SettingTest.py

```diff
@@ -28,6 +29,11 @@ def test_path(self):
                               origin="test" + os.path.sep),
                          os.path.abspath(os.path.join("test", "22")))
 
+    def test_glob(self):
+        self.uut = Setting("key", ".",
+                           origin="test (1)" + os.path.sep + "somefile")
```

Comment:
Wait, I think there might be some problem with windows related to escaping (`C:\project\...` being interpreted as "C:project...` (without the backslash)). I experienced this in an Appveyor test once and it failed. I'm not sure though.